functions/common.py

```diff
@@ -24,3 +25,40 @@ def mod(dividend: float, divisor: float) -> float:
     while dividend < divisor:
         dividend += divisor
     return dividend
+
+
```

Comment:
I created a file called display_output in the PR for our rad->deg conv function. 
I was thinking maybe all of these types of functions should be in that file. 
What do you think?

Reply:
Are you referring to the factorial() function or the decimal_to_binary_integer() and decimal_to_binary_decimal() functions?